nautobot/core/templates/utilities/worker_status.html

```diff
@@ -0,0 +1,138 @@
+{% extends 'base.html' %}
+{% load helpers %}
+
+{% block extra_styles %}
+<style type="text/css">
+    .panel.panel-default {
+        overflow: hidden
+    }
+</style>
```

Comment:
Fixes this:

![image](https://github.com/user-attachments/assets/0b8391b4-cb2f-4fc0-87f0-e8514e4e70cb)

Reply:
It doesn't appear to be negatively impacting anything but it could make the bug where we have tables that overflow out of their parent panel even more of a nuisance.

Reply:
OK OK, you can keep it here. Sorry for the noise. :-)

Reply:
Tracking the global fix with #5948